docs/application/dotnet/guides/app-management/overview.md

```diff
@@ -20,6 +20,10 @@ You can use the following application management features in your .NET applicati
 
   You can retrieve information about the packages installed on the device. You can also monitor for changes in the device packages.
 
+- [Widget Control](wdiget-control.md)
+
+  You can send update requests to the widget applications. You can also listening widget lifecycle events and retrieving details of running instance for the same package widget application.
```

Comment:
Please change to:
You can send update requests to the widget applications. You can also listen widget lifecycle events, and retrieve details of running instance for the same package widget applications.

Reply:
Done